Use less as the default psql pager in container images.
Derived from docker-library/postgres Docker Official Image packaging.
https://github.com/docker-library/postgres

--- a/src/include/fe_utils/print.h
+++ b/src/include/fe_utils/print.h
@@ -18,5 +18,1 @@
-#ifndef __CYGWIN__
-#define DEFAULT_PAGER "more"
-#else
-#define DEFAULT_PAGER "less"
-#endif
+#define DEFAULT_PAGER "less"
